membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -49,71 +48,89 @@ class AttributeController extends Controller with LoggingWithLogstashFields {
   lazy val authenticationService: AuthenticationService = IdentityAuthService
   lazy val metrics = Metrics("AttributesController")
 
+  private def lookup(endpointDescription: String, onSuccessMember: Attributes => Result, onSuccessMemberAndOrContributor: Attributes => Result, onNotFound: Result, doExperiment: Boolean) =
+  {
+    def pickAttributes(identityId: String, prodTest: Boolean)(implicit request: BackendRequest[AnyContent]) = {
+      prodTest match {
```

Comment:
pattern match on a boolean is neater as an if (...)else statement
two nested pattern match on a boolean with the same false expression is an if (... && ...) else